src/main/frontend/components/VenueTile.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const VenueTile = (props) => {
+  const {
+    name,
+    city,
+    state,
+    address,
+    zip,
+    phoneNumber,
+    capacity,
+    description,
+    imgUrl,
+  } = props.venue;
```

Comment:
nice object destructuring :) 

if we're only going to use `name`, `imgUrl`, and `description` below, let's only destructure those attributes, and remove the other 6 above please!